source/get-started.txt

```diff
@@ -70,18 +70,62 @@ its dependencies in your development environment.
          or `Creating New Gradle Builds <https://guides.gradle.org/creating-new-gradle-builds/>`__
          to learn how to set up your project.
 
+   .. step:: Add the {+driver-short+} Bill of Materials
+
+      .. _java-get-started-install-bom:
+
+      In your IDE, create a new `Maven <https://maven.apache.org/>`__ or
+      `Gradle <https://gradle.org/>`__ project. Add the Bill of
+      Materials (BOM) for MongoDB JVM artifacts to your project to
+      organize dependency versions. The BOM simplifies dependency
+      management by ensuring that you maintain consistent and compatible
+      versions of dependencies, such as between the {+driver-short+} and
+      the core driver library. Use the BOM to avoid version conflicts
+      and simplify upgrades.
+      
+      Select from the following :guilabel:`Maven` and :guilabel:`Gradle` tabs
+      to view instructions for adding the BOM for each dependency manager:
+
+      .. tabs::
+      
+         .. tab:: Maven
+            :tabid: maven bom
+            
+            Add the following code to the ``dependencyManagement`` list in your ``pom.xml`` file:
```

Comment:
q: does the Gradle tab need an instruction like this, or should it be obvious to users?

Reply:
Yes I'll add